applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -395,14 +395,24 @@ Since the Matter stack uses one Bluetooth LE connection for commissioning, the m
 Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
 The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``.
 You can increase the number of Bluetooth LE connections if you decrease the size of the Bluetooth LE TX/RX buffers used by the Bluetooth controller, but this will decrease the communication throughput.
-Build the target using the following command in the project directory to enable a configuration that increases the number of Bluetooth LE connections to ``20`` by decreasing the sizes of Bluetooth LE TX/RX buffers:
-
-.. parsed-literal::
-   :class: highlight
-
-   west build -b nrf7002dk_nrf5340_cpuapp -- -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dhci_ipc_EXTRA_CONF_FILE="*absoule_path*/overlay-bt_max_connections_net.conf"
+Build the target using the following variables to enable a configuration that increases the number of Bluetooth LE connections to ``20`` by decreasing the sizes of Bluetooth LE TX/RX buffers:
+
+.. list-table::
+  :header-rows: 1
+
+  * - Build target
+    - Files and Kconfig options to use
+    - Build variables
+  * - ``nrf7002dk_nrf5340_cpuapp``
+    - | * :kconfig:option:`CONFIG_BRIDGED_DEVICE_BT`
+      | * :file:`overlay-bt_max_connections_app.conf`
+      | * :file:`*absolute_path*/overlay-bt_max_connections_net.conf`
```

Comment:
You should probably take out the pipes for the multi-item cells so that the bullet list builds as a bullet list.